docs/it/come-affrontare-l-eta-nel-mondo-it.md

```diff
@@ -41,15 +41,21 @@ Caro lettore, vogliamo dirti che non è così: se in te c'è passione per lo svi
 
 Concludiamo questa sezione dicendoti che non vogliamo convincerti a non diventare manager. Ci sono tante sfide in questo ruolo che sicuramente ti darebbero molte soddisfazioni. Diventa manager, abbandonando pure la parte tecnica e il codice, se ti rivedi in questa figura, se questo ruolo ti appassiona e sei convinto di questa decisione. Siamo certi che se intraprenderai questo percorso con decisione e passione potrai diventare un manager con il quale tutti avranno piacere di collaborare.
 
-# Ruoli tecnici sfidanti
+## Non _solo_ developers
```

Comment:
```suggestion
## Non _solo_ developer
```

Reply:
La s finale tende a pluralizzare in inglese, ma essendo il contenuto in italiano è da rimuovere, vero @Il-Libro-Open-Source/drafting-group ?

Reply:
L'osservazione mi torna.
Attendiamo altri pareri per conferma.